tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2474,3 +2475,184 @@ def test_limit_number_of_fields(self):
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert (
+            response.data["data"][0]["avg_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert (
+            response.data["data"][0]["sum_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+
+    def get_measurement_condition_response(self, query_str, field):
+        query = {
+            "field": ["transaction", "count()"] + (field if field else []),
+            "query": query_str,
+        }
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
+        return response
+
+    def assert_measurement_condition_without_results(self, query_str, field=None):
+        response = self.get_measurement_condition_response(query_str, field)
+        assert len(response.data["data"]) == 0
+
+    def assert_measurement_condition_with_results(self, query_str, field=None):
+        response = self.get_measurement_condition_response(query_str, field)
+        assert len(response.data["data"]) == 1
+        assert response.data["data"][0]["transaction"] == self.transaction_data["metadata"]["title"]
+        assert response.data["data"][0]["count"] == 1
+
+    def test_measurements_conditions(self):
+        self.store_event(self.transaction_data, self.project.id)
+
+        # the conditions only work with integer values at the moment
+        fcp = int(self.transaction_data["measurements"]["fcp"]["value"])
+
+        # ============================== Equality ==============================
+        # The parser only works with integer values in these conditions right now
+        # so we cannot filter for equality on floating point values.
+
+        # ============================== Greater Than ==============================
+        self.assert_measurement_condition_with_results("measurements.fcp:>{}".format(fcp - 1))
+        self.assert_measurement_condition_without_results("measurements.fcp:>{}".format(fcp + 1))
+
+        # ============================== Less Than ==============================
+        self.assert_measurement_condition_with_results("measurements.fcp:<{}".format(fcp + 1))
+        self.assert_measurement_condition_without_results("measurements.fcp:<{}".format(fcp - 1))
+
+        # ============================== Has/!Has ==============================
+        self.assert_measurement_condition_with_results("has:measurements.fcp")
+        self.assert_measurement_condition_without_results("!has:measurements.fcp")
+
+    def test_measurements_aggregation_conditions(self):
+        self.store_event(self.transaction_data, self.project.id)
+
+        # the conditions only work with integer values at the moment
+        fcp = int(self.transaction_data["measurements"]["fcp"]["value"])
+
+        # ============================== percentile ==============================
```

Comment:
Banner style comments are not well used in the application. Could we use a simpler comment?

Reply:
Addressed in 2f7d0a1